docker-local/docker-compose.yml

```diff
@@ -332,6 +335,92 @@ services:
+      - "./dfsp_a/dfsp_a_thirdparty_scheme_adapter.json:/opt/thirdparty-scheme-adapter/config/development.json"
+      - "./docker/wait4:/opt/thirdparty-scheme-adapter/wait4"
+    ports:
+      - "5005:4005"
+    networks:
+      - mojaloop-net
+    depends_on:
+      - dfspa-redis
+    restart: always
+    healthcheck:
+      test: wget -q http://172.17.0.1:5005/health -O /dev/null || exit 1
+      timeout: 3s
+      retries: 10
+      start_period: 10s
+      interval: 30s
+    extra_hosts:
+      - "central-ledger.local:172.17.0.1"
+      - "central-settlement.local:172.17.0.1"
+      - "ml-api-adapter.local:172.17.0.1"
+      - "account-lookup-service.local:172.17.0.1"
+      - "account-lookup-service-admin.local:172.17.0.1"
+      - "quoting-service.local:172.17.0.1"
+      - "moja-simulator.local:172.17.0.1"
+      - "central-ledger:172.17.0.1"
+      - "central-settlement:172.17.0.1"
+      - "ml-api-adapter:172.17.0.1"
+      - "account-lookup-service:172.17.0.1"
+      - "account-lookup-service-admin:172.17.0.1"
+      - "quoting-service:172.17.0.1"
+      - "simulator:172.17.0.1"
+      - "host.docker.internal:172.17.0.1"
+      - "simulator.local:172.17.0.1"
+      - "transaction-request-service:172.17.0.1"
+      - "dfspa-backend:172.17.0.1"
+
+  dfspa-thirdparty-scheme-adapter-outbound:
+    image: "mojaloop/thirdparty-scheme-adapter:v11.15.0"
+    container_name: dfspa-thirdparty-scheme-adapter-outbound
+    environment:
+      - NODE_ENV=development
+    command: sh -c "/opt/thirdparty-scheme-adapter/wait4/wait4.js dfspa-thirdparty-scheme-adapter && npm run start:outbound"
+    volumes:
+      - "./dfsp_a/secrets:/opt/thirdparty-scheme-adapter/secrets"
+      - "./dfsp_a/dfsp_a_thirdparty_scheme_adapter.json:/opt/thirdparty-scheme-adapter/config/development.json"
+      - "./docker/wait4:/opt/thirdparty-scheme-adapter/wait4"
+    ports:
+      - "5006:4006"
+    networks:
+      - mojaloop-net
+    depends_on:
+      - dfspa-redis
+    restart: always
+    healthcheck:
+      test: wget -q http://172.17.0.1:5006/health -O /dev/null || exit 1
+      timeout: 3s
+      retries: 10
+      start_period: 10s
+      interval: 30s
+    extra_hosts:
+      - "central-ledger.local:172.17.0.1"
+      - "central-settlement.local:172.17.0.1"
+      - "ml-api-adapter.local:172.17.0.1"
+      - "account-lookup-service.local:172.17.0.1"
+      - "account-lookup-service-admin.local:172.17.0.1"
+      - "quoting-service.local:172.17.0.1"
+      - "moja-simulator.local:172.17.0.1"
+      - "central-ledger:172.17.0.1"
+      - "central-settlement:172.17.0.1"
+      - "ml-api-adapter:172.17.0.1"
+      - "account-lookup-service:172.17.0.1"
+      - "account-lookup-service-admin:172.17.0.1"
+      - "quoting-service:172.17.0.1"
+      - "simulator:172.17.0.1"
+      - "host.docker.internal:172.17.0.1"
+      - "simulator.local:172.17.0.1"
+      - "transaction-request-service:172.17.0.1"
+      - "dfspa-backend:172.17.0.1"
+
   dfspa-redis:
```

Comment:
no port mapping for Redis? Will it be with conflict with dfspb-redis and other Redis instances dedicated to scheme-adapters, or not?

Reply:
I don't think we need to expose any ports externally.

Reply:
okay, let keep it as it is.